test/integration/fixtures/blocks/core__button__border_radius__deprecated.json

```diff
@@ -3,6 +3,8 @@
 		"name": "core/button",
 		"isValid": true,
 		"attributes": {
+			"tagName": "a",
+			"type": "button",
```

Comment:
For blocks like this, I'm wondering if a `default` is really necessary, because the save and edit functions also set a default if the attribute is undefined.

Reply:
why do the save and edit functions set a default if the block registration has a default? maybe it's all for extra safety?

anything on the server won't get any defaults from a save/edit function, but they will if they have the `block.json`, so I would imagine it's best to keep any defaults _there_ if we're asking

Reply:
Yes, agreed. The point I'm trying to make here is that we should be able to rely on the types and defaults set by the block attribute schema.